eth-contracts/scripts/migrate-contracts.js

```diff
@@ -81,7 +92,7 @@ module.exports = async callback => {
   const libsDirRoot = path.join(getDirectoryRoot(Libs), 'eth-contracts')
   fs.removeSync(libsDirRoot)
 
-  copyBuildDirectory(libsDirRoot + '/ABIs')
+  await copyBuildDirectory(libsDirRoot + '/ABIs')
```

Comment:
should probably do a path join here instead of manual concat even if we are the ones suppling libsDirRoot

Reply:
same in the other places where we do  `x + '/ABIs'`

Reply:
👍